shared/actions/config/index.js

```diff
@@ -30,6 +30,28 @@ export function getConfig (): AsyncAction {
   }
 }
 
+export function getExtendedConfig (): AsyncAction {
+  return dispatch => {
+    return new Promise((resolve, reject) => {
+      const params : config_getExtendedStatus_rpc = {
+        method: 'config.getExtendedStatus',
+        param: {},
+        incomingCallMap: {},
+        callback: (error, extendedConfig) => {
+          if (error) {
+            reject(error)
+          }
+
+          dispatch({type: Constants.extendedConfigLoaded, payload: {extendedConfig}})
+          resolve(extendedConfig)
```

Comment:
it is for things that aren't remote and sometimes you'd like that and it makes logic simpler, for example we rely on that to do our tracker timer stuff: https://github.com/keybase/client/blob/master/shared/actions/tracker.js#L28

And that's fine because it runs on the main window and not a remote component